manuscripts/report.py

```diff
@@ -675,6 +676,22 @@ def period_name(cls, pdate, interval='quarter', offset=None):
 
         return name
 
+    def find_and_replace(self, directory, to_replace, replacement, file_type=None):
```

Comment:
I think this is better a static method, since there is no reference to self in the code. This would also simplify tests.

Reply:
Yeah, that makes sense.